src/main/java/net/neoforged/neoforge/registries/RegistryManager.java

```diff
@@ -96,6 +97,18 @@ public static void postNewRegistryEvent() {
         pendingModdedRegistries = null;
     }
 
+    private static boolean canModifyComponents;
```

Comment:
Nitpick: I'd rather have this in `RegistrationEvents` than in `RegistryManager` which we should probably try to keep for generic registry handling.